create-filter/index.test.ts

```diff
@@ -692,3 +692,41 @@ it('clean filters', () => {
 
   expect(client.log.actions()).toHaveLength(3)
 })
+
+it('doesn\'t subscribe to children  with singleSubscription option', async () => {
```

Comment:
Honestly, I would prefer running most of the tests with this option, to check that we don't lose important functionality